src/App.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import Calculator from './components/Calculator';
+
+
 class App extends React.Component {
```

Comment:
- [ ] [OPTIONAL] Please try to add more commits to the pull requests next time. At least 5 of them make it look professional 🙌🏼